content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -260,6 +255,50 @@ The above update rules apply to regular pod updates, but other pod fields can be
 - **Binding:** The `binding` subresource allows setting the pod's `spec.nodeName` via a `Binding` request.
   This is typically only used by the {{< glossary_tooltip text="scheduler" term_id="kube-scheduler" >}}.
 
+### Pod generation
+
+- The `metadata.generation` field is unique. It will be automatically set by the
+  system such that new pods have a `metadata.generation` of 1, and every update to
+  mutable fields in the pod's spec will increment the `metadata.generation` by 1.
+
+{{< feature-state feature_gate_name="PodObservedGenerationTracking" >}}
+
+- `observedGeneration` is a field that is captured in the `status` section of the Pod
+  object. If the feature gate `PodObservedGenerationTracking` is set, the Kubelet will set `status.observedGeneration`
+  to track the pod state to the current pod status. The pod's `status.observedGeneration` will reflect the
+  `metadata.generation` of the pod at the point that the pod status is being reported.
+
+Different status fields may either be associated with the `metadata.generation` of the current sync loop, or with the
+`metadata.generation` of the previous sync loop. The key distinction is whether a change in the `spec` is reflected
+directly in the `status` or is an indirect result of a running process.
+
```

Comment:
can we add two notices - one for the fact that this does not apply to static pods. And one that node lifecycle controller may set this as well?

Reply:
The node lifecycle controller does not touch `status.observedGeneration`. It only touches pod `conditions[].observedGeneration` which does not apply to this documentation.

If we want to make it a requirement to document more carefully `conditions[].observedGeneration`, let's leave it out of scope of this PR. I'll take it up as a separate task.

Reply:
sure. Makes sense